webpack/services/api.js

```diff
@@ -0,0 +1,13 @@
+import { getApiRequest } from './mock';
```

Comment:
I think ideally we'd have a way to mock out specific URLs and have all others that are not mocked pass directly to the API.  Then when the actual API is available we can simply remove the mocked API from the list of APIs and everything should work.

Perhaps we could accomplish this here by passing through to the real API if the mocked API is not found.

Reply:
I totally agree here. We need a more robust solution. I looked more at the middleware setup with redux and I wonder if we can do something with redux-thunk which allows us to return a function rather than an action directly. Let's talk more on this.

Reply:
Foreman already [requires redux-thunk](https://github.com/theforeman/foreman/blob/develop/package.json#L84) so I think that would be a fine solution.